src/test/fixtures/configuration-differences/expected.txt

```diff
@@ -0,0 +1,230 @@
++     |    |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.71 -> 1.5.0 (*)
++     |    |         \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.3.71 -> 1.5.0
++     |    +--- org.jetbrains.kotlin:kotlin-scripting-jvm:1.5.0
++     |    |    +--- org.jetbrains.kotlin:kotlin-script-runtime:1.5.0
++     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.5.0 (*)
++     |    |    \--- org.jetbrains.kotlin:kotlin-scripting-common:1.5.0 (*)
++     |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.5.0 (*)
++     |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.8 (*)
++     \--- org.jetbrains.kotlin:kotlin-stdlib:1.5.0 (*)
+
++kotlinCompilerPluginClasspathTest - Kotlin compiler plugins for compilation 'test' (target  (jvm))=+\--- org.jetbrains.kotlin:kotlin-scripting-compiler-embeddable:1.5.0
++     +--- org.jetbrains.kotlin:kotlin-scripting-compiler-impl-embeddable:1.5.0
++     |    +--- org.jetbrains.kotlin:kotlin-scripting-common:1.5.0
++     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.5.0
++     |    |    |    +--- org.jetbrains:annotations:13.0
++     |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.5.0
++     |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.8
++     |    |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.71 -> 1.5.0 (*)
++     |    |         \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.3.71 -> 1.5.0
++     |    +--- org.jetbrains.kotlin:kotlin-scripting-jvm:1.5.0
++     |    |    +--- org.jetbrains.kotlin:kotlin-script-runtime:1.5.0
++     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.5.0 (*)
++     |    |    \--- org.jetbrains.kotlin:kotlin-scripting-common:1.5.0 (*)
++     |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.5.0 (*)
++     |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.8 (*)
++     \--- org.jetbrains.kotlin:kotlin-stdlib:1.5.0 (*)
+
++\--- org.jetbrains.kotlin:kotlin-scripting-compiler-embeddable:1.5.0
++     +--- org.jetbrains.kotlin:kotlin-scripting-compiler-impl-embeddable:1.5.0
++     |    +--- org.jetbrains.kotlin:kotlin-scripting-common:1.5.0
++     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.5.0
++     |    |    |    +--- org.jetbrains:annotations:13.0
++     |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.5.0
++     |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.8
++     |    |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.71 -> 1.5.0 (*)
++     |    |         \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.3.71 -> 1.5.0
++     |    +--- org.jetbrains.kotlin:kotlin-scripting-jvm:1.5.0
++     |    |    +--- org.jetbrains.kotlin:kotlin-script-runtime:1.5.0
++     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.5.0 (*)
++     |    |    \--- org.jetbrains.kotlin:kotlin-scripting-common:1.5.0 (*)
++     |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.5.0 (*)
++     |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.8 (*)
++     \--- org.jetbrains.kotlin:kotlin-stdlib:1.5.0 (*)
```

Comment:
something is weird here `=` shouldn't be there and the subtree is duplicated

Reply:
Fixed in https://github.com/JakeWharton/dependency-tree-diff/compare/bcb15a72056c9563e315e2cfa0c1756318ba618f..ba817defa86878ee6962303dc47fae5f5343ff66